ccd-definition/ComplexTypes/CareSupervision/1_SupportingDocuments/SupportingEvidenceBundle-nonprod.json

```diff
@@ -0,0 +1,44 @@
+[
```

Comment:
I think this file doesn't need to be in "nonprod" section. Even if we configure it in prod nothing wrong is going to happen.

Reply:
agree, done.